development-vm/README.md

```diff
@@ -26,7 +26,7 @@ checking for other manufacturers.
   * You have [OSX GCC tools installed](https://github.com/kennethreitz/osx-gcc-installer) (or via XCode)
   * You have [VirtualBox](https://www.virtualbox.org/) installed
   * You have an account on [GitHub.com](https://github.com)
-  * You have [Vagrant 1.7.4](https://releases.hashicorp.com/vagrant/1.7.4/vagrant_1.7.4.dmg)
+  * You have [Vagrant 1.8.6](https://releases.hashicorp.com/vagrant/1.8.6/vagrant_1.8.6.dmg)
```

Comment:
Would be better to say "the latest version of Vagrant" I think and link to the main download page.